### ISSUE TITLE
recurring event does re-appear

### ISSUE THREAD

**Issue by gour** [dagraham/etm-dgraham#4]
Hello,

I've the following event defined:
```
- vaišnavski kalendar (papir) @s 2020-05-25
@i personal
@t computer
@r m
```
Now, I did finish it (few times) with `F`, but it keeps re-appearing in my agenda and here is the shot showing `@h` entries associated with it:
![etm](https://user-images.githubusercontent.com/114592/83945794-5a39cc80-a80d-11ea-8cd3-a722803fec08.png)

The entry in `db.json` looks as:

```
"47": {
   "created": "{T}:20200606T1334A",
   "itemtype": "-",
   "summary": "vaišnavski kalendar (papir)",
   "s": "{D}:20200525",
   "r": [
    {
     "r": "m"
    }
   ],
   "i": "personal",
   "t": [
    "computer"
   ],
   "h": [
    "{T}:20200606T1336A",
    "{T}:20200606T1337A",
    "{T}:20200606T1341A"
   ],
   "modified": "{T}:20200606T1341A"
  }
```
so I wonder why is `Fınished task not 'recycled, but keeps re-appearing?

Sincerely,
Gour

**Comment by dagraham** [comment]
Fixed in 4.5.2  A curious error that only occurred in repeating tasks with dates rather than datetimes as instances and in timezones with positive offsets relative to UCT.

**Comment by gour** [comment]
> Fixed in 4.5.2

I confirm that the fix is working here.

>  A curious error that only occurred in repeating tasks with dates rather than datetimes as instances and in timezones with positive offsets relative to UCT.

I'm glad I was able to contribute  a bit to `etm` by providing some corner cases... :smile:

**Comment by dagraham** [comment]
👍😊
